common/lib/capa/capa/tests/test_files/extended_hints_multiple_choice_with_img.xml

```diff
@@ -0,0 +1,34 @@
+<problem>
+<p>(note the blank line before mushroom -- be sure to include this test case)</p>
+<p>Select the fruit from the list</p>
+<multiplechoiceresponse>
+  <choicegroup label="Select the fruit from the list" type="MultipleChoice">
+    <choice correct="false">Mushroom
+        <choicehint label="">Mushroom <img src="#" ale="#"/>is a fungus, not a fruit.
+        </choicehint>
+    </choice>
+    <choice correct="false">Potato</choice>
+    <choice correct="true">Apple
+        <choicehint label="OUTSTANDING">Apple is indeed a fruit.
+        </choicehint>
+    </choice>
+  </choicegroup>
+</multiplechoiceresponse>
+<p>Select the vegetables from the list</p>
```

Comment:
You're right on both accounts, it was ambiguous. I've changed the test to be more concise and obvious that it was testing hints which contain html code (imgs and such)